python/src/ecole/_combinatorial_auction_generator.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+
+from pyscipopt import Model

```

Comment:
Could we lazy load PySCIPOpt by moving this import statement inside of `generate_instance`?
PySCIPOpt is optional dependency, but because all modules are imported, this would raise an error.